src/avgScrabbleScoreFromFile.js

```diff
@@ -11,14 +11,16 @@ let avgScrabbleScore = require('./avgScrabbleScore');
  * sample-words.txt.
  */
 function avgScrabbleScoreFromFile(fileName) {
-  // This is your job. :)
+  let wordsArray = fs.readFileSync(`../src/${fileName}`, 'utf-8').split('\n'); //newline
```

Comment:
1. `../src/` is redundant since the file itself is in the `src` directory. So `../src/` means "the `src` directory in the parent directory", which is the very directory that the file is in! You can do `fs.readFileSync(fileName, 'utf-8')`.
2. There's a small off-by-one error here. Test it with a simple file that contains a single letter and see if you can't find the problem.